index.html

```diff
@@ -1,8 +1,73 @@
    <body>
-      <!-- Write your HTML in this file! -->
+      <header>
+         <article>
+            <h1 id="mainHeading">Now We Are Six</h1>
+           <div class="extra-padding">
+            <p id= "testP" class="funParagraph" >My beautiful Rosemarie was born in April 2017. 
+               Three days of labor and she came out on her own time. 
+               She just turned five and I started thinking about her sixth birthday and this poem.
+               My aunt gave me A.A. Milne's book when I turned six, and my partner and I will do the same for Rosemarie.
+               She continues to grow and develop her own personality, as we all do beyond the age of six. 
+               I love her empathy, her curiosity, her bravery, her independence. 
+               She is my love bug, and I dedicate his page to her.</p>
+            <p id="testP" class="align-right funParagraph">To my Rosemarie <br>- Love, your Mommy.
+            </p>
+         </div>
+         </article>
+      </header>
+
+      <main>
+         <article>
+            <h2>&quot;When I was One,
+               I had just begun.&quot;</h2>
+            <img src="C:\Users\a\OneDrive\WebDevThings\Graded_Assignments\js-assignment-4-html-me-something-saramuseo\images\Year1Rosemarie.jpg" 
+            alt="One year old Rosemarie laying down reading book.">
+         </article>
+
+         <article>
+            <h2> &quot;When I was Two,
+               I was nearly new.&quot;</h2>
+            <img src="C:\Users\a\OneDrive\WebDevThings\Graded_Assignments\js-assignment-4-html-me-something-saramuseo\images\Year2Rosemarie.jpg" 
+            alt="Two year old Rosemarie playing in museum.">
+         </article>
+
+         <article>
+            <h2>&quot;When I was Three
+               I was hardly me.&quot;</h2>
+            <img src="C:\Users\a\OneDrive\WebDevThings\Graded_Assignments\js-assignment-4-html-me-something-saramuseo\images\Year3Rosemarie.jpg" 
+            alt="Three year old Rosemarie at the beach.">
+         </article>
+
+         <article>
+            <h2>&quot;When I was Four,
+               I was not much more.&quot;</h2>
+            <img src="C:\Users\a\OneDrive\WebDevThings\Graded_Assignments\js-assignment-4-html-me-something-saramuseo\images\Year4Rosemarie.jpeg" 
+            alt="Four year old Rosemarie fishing.">
+         </article>
+
+         <article>
+            <h2>&quot;When I was Five,
+               I was just alive.&quot;</h2>
+            <img src="C:\Users\a\OneDrive\WebDevThings\Graded_Assignments\js-assignment-4-html-me-something-saramuseo\images\Year5Rosemarie.jpg" 
+            alt="Five year old Rosemarie brushing teeth while standing on bag of toilet paper.">
+         </article>
+
+         <article>
+            <h2>&quot;But now I am Six,
+               I'm as clever as clever, <br>
+               So I think I'll be six now for ever and ever.&quot;</h2>
+            <p id="square"></p> 
+         </article>
+      </main>
```

Comment:
I think you did a really great job. This file is extremely readable and well formatted. Also, Rosemarie seems like such a cool kid